storage/postgres.go

```diff
@@ -1156,93 +1197,49 @@ func (p *PostgresStorage) RetrieveCredBySubTypeAndAcct(scredType pb.SubCredType,
 	return creds, rows.Err()
 }
 
-func (p *PostgresStorage) GetTrackedRepos() (*pb.AcctRepos, error) {
+func (p *PostgresStorage) GetVCSTypeFromAccount(account string) (pb.SubCredType, error){
+	var bad pb.SubCredType
```

Comment:
The implicitness of this var tripped me up a little, since it didn't seem to be providing any value. This should be a little more explicit. Maybe even declared `const`

```suggestion
	var bad pb.SubCredType = pb.SubCredType_NIL_SCT
	
	//  I'm not sure if this form is compilable 
	// const pb.SubCredType bad = pb.SubCredType_NIL_SCT
```